test/include.mk

```diff
@@ -125,7 +125,8 @@ PATTERN_OPTS = --pattern "$$(cat $*.k)"
 
 ### SCRIPTS
 
-test-%.sh.out: $(KORE_EXEC) $(TEST_DIR)/test-%.sh $(TEST_DIR)/test-%-*
+.SECONDEXPANSION:
+test-%.sh.out: $(KORE_EXEC) $(TEST_DIR)/test-%.sh $$(wildcard $(TEST_DIR)/test-$$*-*)
```

Comment:
`SECONDEXPANSION` is kinda tricky; I think I would prefer just to remove the default dependency on `$(TEST_DIR)/test-%-*` and leave it to specific tests to declare their dependencies in a downstream `Makefile`.